coin_prices_from_coingecko/output.go

```diff
@@ -9,9 +9,9 @@ import (
 )
 
 type Result struct {
-	Success bool
-	Error   string
-	Value   any
+	Success bool   `json:"success"`
+	Error   string `json:"error,omitempty"`
+	Value   any    `json:"value,omitempty"`
```

Comment:
added some json tags to trend away from go-case json fields.

Reply:
I think the omitempty could get confusing to users, when they don't see the field in the output. I think this works in our case because you're only testing for success in coin_prices_from_coingecko.txtar. However, omitting these values would change the output on README.md line 44, no?

Reply:
hm, good point. I like the ergonomics of the omitempty behavior here as a dev, but it has changed a user facing thing and was not a necessary change. I can change it back if you think it is confusing behavior to users.

Reply:
I removed the addition of omitempty